src/gallium/drivers/lima/lima_context.h

```diff
@@ -200,6 +209,9 @@ struct lima_context {
    #define pp_plb_offset(i, n)       \
       (pp_plb_offset_start + i * ((pp_stack_offset - pp_plb_offset_start) / n))
 
+   struct lima_bo *tex_descs;
```

Comment:
Why not add this desc to pp_buffer instead of creating a new one?

Reply:
This bo content may need be changed for every draw, same as pp_buffer attribute/uniform, not like shader bo can persist across draws/frames. Reduce bo num is preferred.